dist/flatpak/python3-attrs.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "python3-attrs",
+    "buildsystem": "simple",
+    "build-commands": [
+        "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"attrs\" --no-build-isolation"
+    ],
+    "sources": [
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/6a/21/5b6702a7f963e95456c0de2d495f67bf5fd62840ac655dc451586d23d39a/attrs-24.2.0-py3-none-any.whl",
+            "sha256": "81921eb96de3191c8258c199618104dd27ac608d9366f5e35d011eae1867ede2"
+        }
+    ]
+}
```

Comment:
Please add an empty new line in this and the other json file

Reply:
I'd rather not, those are autogenerated files by https://github.com/flatpak/flatpak-builder-tools/blob/master/pip/flatpak-pip-generator

Reply:
As a side note, we should be able to drop the dependency on `python3-attrs` once https://gitlab.freedesktop.org/libinput/libei/-/merge_requests/302 lands in libei.